README.md

```diff
@@ -111,19 +111,15 @@ the target tag should be `v1.5.2-2`, `v1.5.2-3`, and so on.
 
 It is also possible to watch and automatically retag new tagged releases in the upstream repository.
 To do this, specify a pattern for the image in the `images.yaml` configuration file.
-Each pattern must be a valid Go regular expression (you can try it out [here](https://regex-golang.appspot.com/assets/html/index.html)),
+Each pattern must be a valid semver expression (you can read about it [here](https://github.com/Masterminds/semver)),
```

Comment:
Maybe mention that the `v` is optional. Or for consistency we could require that it is there.